static/app/utils/feedback/types.tsx

```diff
@@ -10,9 +10,10 @@ export type FeedbackIssue = Overwrite<
     metadata: {
       contact_email: null | string;
       message: string;
-      name: string;
+      name: null | string;
       title: string;
       value: string;
+      initial_priority?: number;
```

Comment:
what's this for?

Reply:
needed to make the types work. the real response in the browser has it so i think it was either recently added to issues, or we just missed it when we did the types the first time around.